src/lib/operators/join_hash/join_hash_steps.hpp

```diff
@@ -70,15 +68,35 @@ template <typename T>
 struct RadixContainer {
   std::shared_ptr<Partition<T>> elements;
   std::vector<size_t> partition_offsets;
+
+  // bit vector to store NULL flags
+  std::shared_ptr<std::vector<bool>> position_is_null_value;
 };
 
-template <typename T, typename HashedType>
+inline std::vector<size_t> determine_chunk_offsets(std::shared_ptr<const Table> table) {
```

Comment:
Why inline?

Reply:
To avoid duplicate symbol errors when libhyrise and the test include the header file (there is no cpp file).

Reply:
Anonymous namespace?

Reply:
I have no preferences apart from other functions using `inline` already.

Reply:
Ah, nevermind. In a header, anonymous namespaces would violate the ODR.